corehq/apps/reports/views.py

```diff
@@ -468,39 +466,67 @@ def user_id(self):
 
 @login_and_domain_required
 @datespan_default
-def email_report(request, domain, report_slug, report_type=ProjectReportDispatcher.prefix, once=False):
+def email_report(request, domain, report_slug, dispatcher_klass=ProjectReportDispatcher, once=False):
```

Comment:
Why misspell "class"?
```suggestion
def email_report(request, domain, report_slug, dispatcher_class=ProjectReportDispatcher, once=False):
```
Obviously it will need to be renamed in other places as well.

Reply:
Right, with the `dispatcher_` prefix, it won't be a reserved word...